src/cascadia/TerminalSettingsModel/ActionArgs.h

```diff
@@ -782,6 +865,18 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             h.write(ContentArgs());
             return h.finalize();
         }
+        winrt::Windows::Foundation::Collections::IVectorView<Model::ArgDescriptor> GetArgDescriptors()
+        {
+            return _ContentArgs.as<NewTerminalArgs>()->GetArgDescriptors();

```

Comment:
for every place where you `as<NewTerminalArgs>` you can `as<IActionArgsDescriptorAccess>` to future-proof yourself